doc/sedcli.8

```diff
@@ -81,6 +81,15 @@ Changes the lock state for the device. Following access modes are available:
 .IP "\fB\-P -d <device>\fR or \fB\-\-set-password --device <device>\fR"
 Updates password for Admin1 authority in Locking SP.
 
+.IP "\fB\-M -d <device> -e <enable> -m <done>\fR or \fB\-\-mbr-control --device <device> --enable {TRUE|FALSE} --done {TRUE|FALSE}\fR"
```

Comment:
Please indicate which parameters are optional and selection list vs placeholder for arbitrat value. Here is my recommendation:
-M -d \<device\> [-e {TRUE|FALSE}]  [-m {TRUE|FALSE}] or --mbr-control --device \<device\>
       [--enable {TRUE|FALSE}] [--done {TRUE|FALSE}]